pkgs/top-level/all-packages.nix

```diff
@@ -10198,6 +10198,7 @@ with pkgs;
 
   v8 = callPackage ../development/libraries/v8 {
     inherit (python2Packages) python gyp;
+    libtool = darwin.cctools;
```

Comment:
This name seems a bit confusing because it will get confused with GNU lib tool. How about just calling it "cctools" instead?